motion/p5js/connected-elements/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Marius Claßen
+author-url: https://github.com/mariusclassen
+title: Air Dancer
+tags: [P5.js, motion, law of connected elements]
```

Comment:
The tag for the law is:  

    Law of Connected Elements

see https://interface.fh-potsdam.de/gestalten-in-code/law-of/connected-elements/